pandas/util/_decorators.py

```diff
@@ -90,9 +104,9 @@ def wrapper(*args, **kwargs):
 def deprecate_kwarg(
     old_arg_name: str,
     new_arg_name: Optional[str],
-    mapping: Optional[Union[Dict, Callable[[Any], Any]]] = None,
+    mapping: Optional[Union[Dict[Any, Any], Callable[[Any], Any]]] = None,
```

Comment:
I think OK just to do `Dict` instead of `Dict[Any, Any]`; same thing but more readable

Reply:
it is the same thing but would fail --disallow-any-generics check.

although readability is important, i don't think the benefits of precise type checking can be understated.